pkg/integration/tests/branch/delete.go

```diff
@@ -31,13 +39,62 @@ var Delete = NewIntegrationTest(NewIntegrationTestArgs{
 			SelectNextItem().
 			Press(keys.Universal.Remove).
 			Tap(func() {
-				t.ExpectPopup().Confirmation().
-					Title(Equals("Delete branch")).
-					Content(Contains("Are you sure you want to delete the branch 'branch-one'?")).
+				t.ExpectPopup().
+					Menu().
+					Title(Equals("Delete branch 'branch-three'?")).
+					Select(Equals("l Delete local branch")).
```

Comment:
```suggestion
					Select(Contains("Delete local branch")).
```
I prefer excluding the keybinding because we could change how we display those over time and that would break the test unnecessarily